src/crimson/os/seastore/journal/circular_bounded_journal.h

```diff
@@ -22,8 +22,7 @@
 
 namespace crimson::os::seastore::journal {
 
-constexpr rbm_abs_addr CBJOURNAL_START_ADDRESS = 0;
-constexpr uint64_t CBJOURNAL_MAGIC = 0xCCCC;
+constexpr rbm_abs_addr CBJOURNAL_START_ADDRESS = 4096;
```

Comment:
The RBMDevice defines the start of the journal, so load from `RBMDevice::get_journal_start()`.

Reply:
Done.